tests/functional/testplan/runnable/interactive/test_api.py

```diff
@@ -65,32 +62,39 @@ def test_parametrized(self, env, result, val):
 @pytest.fixture
 def plan(tmpdir):
     """Yield an interactive testplan."""
-    plan = testplan.TestplanMock(
-        name=six.ensure_str("InteractiveAPITest"),
-        interactive_port=0,
-        interactive_block=False,
-        exporters=[XMLExporter(xml_dir=tmpdir / "xml_exporter")],
-    )
 
-    logfile = tmpdir / "attached_log.txt"
-    logfile.write_text(
-        "This text will be written into the attached file.", encoding="utf8"
-    )
+    with patch(
```

Comment:
Does this fix interactive mode tests in test_plan_top.py? That's great but why...? ;)

Reply:
the pytest environment is really tricky when it comes to the special logic in reloader to build the dependency tree of imported modules, and we do not really test the reloader here, so mocking it out  we do not need to solve this for these cases. We achieve this by the constructor returning None, which means this interactive mode could not do hot reloads.